modules/ROOT/pages/workday/workday-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,255 @@
+|===
+|From Version | To Version
+|11.1.2 |12.0.0
+|===
+
+== Changes in This Release
+
+This release contains the following changes:
+
+* Upgraded the Workday API to 34.0
+* Added the `Student Core` operation, which enables you to invoke the Workday Student Core Web Service
+* Replaced the `On New Objects` input source with the `On Modified Objects` input source, which polls data more efficiently. For more information, see <<on-modified-objects,On Modified Objects>>.
+* Changed all operations so that they return headers as attributes
+* Added the ability to create response timeouts for the input source (`On Modified Objects`) and all connector operations by using the new `Response Timeout` and `Response Timeout Unit` fields.
+* Split the connection timeout fields into value and time unit fields. For more information, see <<changes-in-connection-types,Changes in Connection Types>>.
+* Removed the `blocking` field from the `Reconnect` and `Reconnect Forever` objects
+* Removed the `WORKDAY:UNKNOWN` error type
+
+[[changes-in-connection-types]]
+== Changes in Connection Types
+
+The following fields are split into value and time unit fields:
+
+// [cols="50,50"]
+[%header%autowidth.spread]
+|===
+|Field in v11.x | Replaced By
+|Connection Idle Timeout
+|Connection Idle Timeout and Connection Idle Timeout Unit
+|Connction Timeout |Connection Timeout and Connection Timeout Unit
+|===
+
+[[on-modified-objects]]
+== On Modified Objects Input Source
+
+The `On New Objects` input source in Workday Connector version 11x polled data in a Workday resource from the beginning of the specified time interval to the present. It did this for each worker in the Workday instance, which caused large time inefficiencies when the instance contained thousands of workers.
+
+The new `On Modified Objects` input source, instead, always polls the latest data in the specified time interval. After one poll iteration, the input source sets the earliest moment as the previous latest moment and the latest moment as the current date and time. These changes to polling make the `On Modified Objects` input source much more efficient than the `On New Objects` input source.
+
+You can use the `On Modified Objects` input source with the following web services and operations:
+*Is the following list correct? I modified it based on the changes to the Reference Guide.*
+
+* `Human Resources Web Service`
+** `Get Job Profiles`
+* `Recruiting` Web Service
+** `Get Evergreen Requisitions`
+** `Get Job Requisitions`
+* `Staffing` Web Service
+** `Get Workers`
+** `Get Positions`
+
+*Please make sure this wording is correct.*
+The following example shows how to use the `On Modified Objects` input source. In this example, the `On Modified Objects` input source listens for worker-related events on the `/put-worker` path and initiates the app when a new worker is added. The app then creates a new position, hires the worker, and logs information about the new hire to the Anypoint Studio console.
```

Comment:
The flow does following: 
- We use te On Modified Object input source to listen all event which regards to Worksers (Creating, Modifing, Deleting).
- Then we have flow /put-worker
Use case: We open url localhost:8081/put-worker which creates a new position and then hire new worker to the created position. In a certain time (depends on fixed frequency) our source the On Modified Objects recives message that we;ve hired the new worker. The triger/listener starts second flow which logs whole information from server.